.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "minigeth"]
+	path = minigeth
+	url = git@github.com:norswap/minigeth.git
```

Comment:
`norswap`?

Reply:
I'm well aware! That's why isn't merged yet. I will switch it it over to the official repo at https://github.com/ethereum-optimism/minigeth, but before that I want to rebase it on top of the latest version of Bedrock's execution engine (it was updated while I was working on this).